src/Compilers/CSharp/Portable/Binder/LookupResult.cs

```diff
@@ -191,7 +191,13 @@ internal static SingleLookupResult NotTypeOrNamespace(Symbol symbol, DiagnosticI
         internal static SingleLookupResult NotTypeOrNamespace(Symbol unwrappedSymbol, Symbol symbol, bool diagnose)
         {
             // TODO: determine correct diagnosis 
-            var diagInfo = diagnose ? new CSDiagnosticInfo(ErrorCode.ERR_BadSKknown, unwrappedSymbol, unwrappedSymbol.GetKindText(), MessageID.IDS_SK_TYPE.Localize()) : null;
+            object sym = unwrappedSymbol;
+            // for CS0118 we dont want to display the type, but the name
+            if (unwrappedSymbol.Kind == SymbolKind.Parameter && diagnose)
```

Comment:
@DustinCampbell I dont really have a problem on what is decided to be displayed. You guys sort that out. The really issue is #7204 in terms of getting consistent test expectations. This specific case just pointed out the issue. But for now, `FormattedSymbol` seems a good way to 'override' symbol display to work in the test suite's favor. (I am not saying the tests are broken, but they are constructed in an incorrect way, which will probably bite again later).